recipes/dtc/meta.yaml

```diff
@@ -0,0 +1,119 @@
+{% set version = '1.6.1' %}
+{% set major_ver = version.split(".")[0] %}
+
+package:
+  name: dtc-package
+  version: {{ version }}
+
+source:
+  url: https://git.kernel.org/pub/scm/utils/dtc/dtc.git/snapshot/dtc-{{ version }}.tar.gz
+  sha256: 6401c9a0f577a270df4632bf0f3e5454ccc7a5ca3caefa67a3e1c29c9c6b8c60
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - bison
+    - flex
+    - make
+    - swig
+    - pkg-config
+  host:
+    # Python bindings
+    - python {{ python }}
+    # Optional yaml support
+    - yaml
+  run:
+    - python {{ python }}
+
+outputs:
+  - name: dtc
+    script: install.sh   # [not win]
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - make
+      host:
+        - {{ pin_subpackage("libfdt", exact=True) }}
+        - yaml
+      run:
+        - {{ pin_subpackage("libfdt", max_pin="x.x") }}
+    test:
+      commands:
+        - dtc --help
+    about:
+      summary: 'Linux Kernel Device Tree Compiler (dtc)'
+      description: |
+       Device Tree Compiler (dtc) toolchain for working with device
+       tree source and binary files
+      license: GPL-2.0-or-later
+      license_family: GPL2
+      license_file: 
+        - GPL
+        - README.license
+  - name: pylibfdt
+    script: install.sh   # [not win]
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - make
+        - swig
+      host:
+        - python {{ python }}
+      run:
+        - python {{ python }}
```

Comment:
pylibfdt statically links to libfdt? Is there a compile option to link dynamically?

Reply:
The extension links against `libfdt.a` when creating (for example) `lib/python3.8/site-packages/_libfdt.cpython-38-x86_64-linux-gnu.so`.  So, the pylibfdt package, does not depend at runtime on the libfdt package.  I don't see a way to change the linkage but I'm not an expert when it comes to building python extensions.

Reply:
Lemme look into this whilst you fix the broken OSX build. Otherwise, we'll just ship it statically linked.